scripts/check-qemu-install

```diff
@@ -396,7 +423,147 @@ try:
     #################################################
     # Executing test-suite
     #################################################
-    if args.raid:
+    if args.tpmtest:
+        def verify_mount():
+            # Verify encrypted and mounted
+            c.sendline('mount | grep vyos_config')
+            c.expect('/dev/mapper/vyos_config on /config.*')
+            c.expect(op_mode_prompt)
+
+        def verify_config():
+            # Verify encrypted config is loaded
+            c.sendline('show config commands | cat')
```

Comment:
Is there a specific reason why we can't use the `vyos.config`'s `returnValue` here? And should we be looking for these options rather than for something really distinctive, e.g., a hostname designed for this test like `vyos-tpm-test`?